common/src/main/java/com/wynntils/wynn/objects/SpellType.java

```diff
@@ -39,6 +41,19 @@ public enum SpellType {
     THIRD_SPELL(ClassType.None, 3, "3rd Spell", 0, 0),
     FOURTH_SPELL(ClassType.None, 4, "4th Spell", 0, 0);
 
+    public static final boolean SPELL_RIGHT = true;
+    public static final boolean SPELL_LEFT = false;
+
+    private static final boolean[] RLR = {SPELL_RIGHT, SPELL_LEFT, SPELL_RIGHT};
+    private static final boolean[] RRR = {SPELL_RIGHT, SPELL_RIGHT, SPELL_RIGHT};
+    private static final boolean[] RLL = {SPELL_RIGHT, SPELL_LEFT, SPELL_LEFT};
+    private static final boolean[] RRL = {SPELL_RIGHT, SPELL_RIGHT, SPELL_LEFT};
+    // Archer only
+    private static final boolean[] LRL = {SPELL_LEFT, SPELL_RIGHT, SPELL_LEFT};
+    private static final boolean[] LLL = {SPELL_LEFT, SPELL_LEFT, SPELL_LEFT};
+    private static final boolean[] LRR = {SPELL_LEFT, SPELL_RIGHT, SPELL_RIGHT};
+    private static final boolean[] LLR = {SPELL_LEFT, SPELL_LEFT, SPELL_RIGHT};
```

Comment:
Pretty sure QuickCastFeature has these too

Reply:
I think Donkey has been struggling enough with this feature. I suggest we accept some duplication, and leave the unification of totem tracking spell handling with quick cast spell handling as a separate issue. I can probably volunteer to handle it.

Reply:
Yeah I was just making a note, similarly to yours about spell model. No need to handle it now

Reply:
Yea, will refactor QuickCastFeature to use all the new Spell stuff added in this PR, after this is merged